bot/cogs/assignable_roles_cog.py

```diff
@@ -197,6 +198,9 @@ async def remove_role(self, ctx, role: discord.Role):
 
     @roles.command(pass_context= True, aliases= ['create'])
     @commands.has_guild_permissions(administrator = True)
+    @ext.long_help('Command to add a role as assignable in the current guild')
+    @ext.short_help('Marks a role as user assignable')
+    @ext.example('roles add #SomeExampleRole')
```

Comment:
Shouldn't it use an @ instead of # for roles?

Reply:
same for line 216

Reply:
ope yes, i did this after the designated channel helps and just mindlessly did it, good catch